docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>There is <strong>a lot</strong> of repetition between this section and</p><p>https://colab.sandbox.google.com/github/BertrandRdp/flax/blob/master/docs/notebooks/jax_for_the_impatient.ipynb#scrollTo=S2BcA8wm2_FW</p><p> </p><p>Why don't you ask readers to go to that other Colab (you can have a link like the above to the Colab on the master branch) instead of repeating it here? Also, the code below has not really a lot to do with Flax, so having all this content in "Flax basics" might be confusing and distract from the essentials.</p><p> </p><p>(not adding more comments on this section)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>The point initially was to walk people through linear regression by adding more stuff in there along the way. So here we still have a basic regression with a Flax model that shows something.</p><p> </p><p>But the suggestion is good, I'll probably hide the redundant parts, and people will be able to expand it if needed. I'll find something.</p>